features/networking/ovn.feature

```diff
@@ -0,0 +1,25 @@
+Feature: OVN related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-32205
+  @admin
+  Scenario: Thrashing ovnkube master IPAM allocator by creating and deleting various pods on a specific node
+    Given the env is using "OVNKubernetes" networkType
```

Comment:
It looks like this step is the only step that requires admin?  Would this test also work on SDN?  Then we could remove the step and not require admin maybe.  Maybe it's not worth it.

Reply:
@rbbratta This is OVN specific as scenario title suggests.